### PR TITLE
Adding refresh of dashboard after deletion of event

### LINKED ISSUE #2483: Refresh Dashboard once event has been deleted
When a user deletes their own event the dashboard still shows the event even though it has been deleted. 

Expected: The dashboard page should refresh after the event is deleted to reflect the status on the site.

Compare: https://next.eventyay.com/events/live

### PR DESCRIPTION
**Pull request by kushthedude**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:

Refreshes dashboard after deletion of event.
#### Changes proposed in this pull request:

- Added *refreshRoute* after success of deletion of event

Fixes #2483

### CODE REVIEW

**Comment by kushthedude** [comment]
@mariobehling Review please

File: app/controllers/events/list.js [hunk 1]
```diff
@@ -75,6 +75,7 @@ export default Controller.extend({
       })
         .then(() => {
           this.notify.success(this.get('l10n').t('Event has been deleted successfully.'));
+          this.send('refreshRoute');
```
Comment (CosmicCoder96) [inline app/controllers/events/list.js]: Where is this `refreshRoute` action defined? @kushthedude

Comment (kushthedude) [reply app/controllers/events/list.js]: I think it was defined in route , Let me give a look will push commit if not defined @CosmicCoder96

**Comment by kushthedude** [comment]
@CosmicCoder96 Please review

**Comment by kushthedude** [comment]
@CosmicCoder96 Did the changes

**Pull request merged**